src/app/js/admin/api/job.js

```diff
@@ -5,9 +5,9 @@ export const getUserLocalStorageInfo = () =>
     JSON.parse(window.localStorage.getItem('redux-localstorage').user);
 
 const getJobLogs = (queue, jobId) => {
-    const state = getUserLocalStorageInfo();
+    const userInfo = getUserLocalStorageInfo();
 
-    const request = getJobLogsRequest(state, queue, jobId);
+    const request = getJobLogsRequest(userInfo, queue, jobId);
```

Comment:
It's not clearer to me what's in this object and why we send it all

Reply:
Looking at the function quickly, it seems we only use the token within the user info, why not send this only?